### PR TITLE
Article about Elasticon

### PR DESCRIPTION
**Pull request by quentinfayet**

### CODE REVIEW

**Comment by pCyril** [comment]
+1

**Comment by tejerka** [comment]
+1

File: _posts/2015-11-06-elasticon-paris-2015-we-were-there.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "Elasticon Paris 2015 : We were there !"
+author: reputation_vip
+modified: 2015-11-06
+excerpt: "ReputationVIP were present at the Elasticon Paris 2015, here is what we thought about it!"
+tags: [blend, blendwebmix, conférence, lyon]
```
Comment (tomlelievre) [inline _posts/2015-11-06-elasticon-paris-2015-we-were-there.md]: Je pense que les tags "blend" et "blendwebmix" sont en trop ici.

**Comment by tomlelievre** [comment]
+1

File: _posts/2015-11-06-elasticon-paris-2015-we-were-there.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+and so he decided to build an abstraction layer on top of it. Elasticsearch was born. Fun facts, when he released the
+first version of Elasticsearch (still in development), Shay Banon decided to tag it as the version 0.4, instead of
+0.1, to give credit to his product.
+
+And so came some more numbers. 35,000 is the total number of users who are part of the Elastic community. Elastic
+has nearly 120 user groups, spread around the world's biggest cities. All projects of the ELK stacks put together
+count more than 32,000 commits! Until now, Elasticsearch has been downloaded more than 35 million times!
+
+Then began the list of the biggest organization and companies that are using Elastic products. Wikimedia foundation
+(Wikipedia) is having a transition to work run under Elasticsearch. Also, Mozilla is using the ELK stack on their
```
Comment (obalais) [inline _posts/2015-11-06-elasticon-paris-2015-we-were-there.md]: > is having a transition to work run under Elasticsearch

Work run?

File: _posts/2015-11-06-elasticon-paris-2015-we-were-there.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+security tool, named as MozDef. The NASA also use the ELK stack to collect and analyze data from martian rovers.
+
+Finally, Shay Banon told us about a funny fact, his first customer on Elastic's IRC channel, Clinton Gormley. As Shay
+was dreaming of his first customer, he imagined amazing projects would use "sexy" language in Elasticsearch, came along
+Clinton, working on a project about funerals, with Perl. As the time goes, Clinton became an priceless member
```
Comment (obalais) [inline _posts/2015-11-06-elasticon-paris-2015-we-were-there.md]: > became a priceless

**Comment by obalais** [comment]
+1

**Pull request merged**